### DISCUSSION TITLE
How can I declare the absolute path of my .msi instead of do a "cd" to use the AppCast Generator?

### DISCUSSION THREAD

**Discussion by anxofcostas** [NetSparkleUpdater/NetSparkle#411] [Q&A]
Hello!

I'm doing a modification to can use NetSparkle with local resources. I've find a problem because I want that, inside of XML file, would be the whole local path at URL field. To do this, I need to introduce the whole local path to my netsparkle-generate-appcast command when I would run it. If there is another solution you can tell me it too.

Can you help me? Thanks!

**Comment by Deadpikle** [comment]
Local paths aren't working right in the app cast generator for some reason that I have not yet been able to research or look into. See #399.
